testproxy/services/close-client.js

```diff
@@ -15,14 +15,17 @@
 
 const normalizeCallback = require('./utils/normalize-callback.js');
 
+const v2 = Symbol.for('v2');
+
 const closeClient = ({clientMap}) =>
   normalizeCallback(async rawRequest => {
     const request = rawRequest.request;
     const {clientId} = request;
     const bigtable = clientMap.get(clientId);
 
     if (bigtable) {
-      bigtable.close();
+      await bigtable[v2].close();
```

Comment:
I actually think this line might not be necessary.

Reply:
Good point but I just confirmed it's necessary. I tried running the tests without closing the v2 instance (removing this line) and tests fail again without it.

My understanding is that veneer and v2 are completely independent instances so it makes sense that it needs to be closed separately.

Reply:
Got it. If that is the case then it looks like the code contains a client bug because closing the veneer is supposed to close the clients on the v2 layer. Oh, but the client never gets initialized because we never make calls with the veneer layer and clients are initialized lazily. I suspect though that when we use the veneer layer for the `mutateRow` calls when `mutateRow` is exposed then we will not need this line anymore.

Thanks for checking.